requirements.txt

```diff
@@ -16,31 +16,31 @@ PyYAML==5.4.1
 ## The following requirements were added by pip freeze:
 appdirs==1.4.4
 astroid==2.5.1
-botocore==1.20.22
+botocore==1.20.25
 click==7.1.2
 contextlib2==0.6.0.post1
 decorator==4.4.2
 gitdb==4.0.5
-GitPython==3.1.14
-importlib-metadata==3.7.0
+GitPython==3.1.11
+importlib-metadata==3.3.0
 jmespath==0.10.0
```

Comment:
wonder why some of these were downgraded?

Reply:
Good question, I can re-run `make deps-update` but I suspect maybe some of these later libraries got pulled for some reason?